atq.json

```diff
@@ -0,0 +1,1099 @@
+{ "type_definitions": 
```

Comment:
Are these three files meant to be committed?

Reply:
removed